src/guide/admonitions.rst

```diff
@@ -0,0 +1,58 @@
+Admonitions
+===========
+
+Admonitions call attention to content by placing it in a box with a label. RST provides several directives for creating admonitions with specific titles and styles, as well as a generic admonition with a title specified by the author.
+
+==================  ============================
+Admonition          Directive
+==================  ============================
+Note                ``.. note::``
+Warning             ``.. warning::``
+Tip                 ``.. tip::``
+Question            ``.. admonition:: Question``
+Example             ``.. example:: Example``
+Examples             ``.. example:: Examples``
+Try It!             ``.. admonition:: Try It!``
+Fun Fact            ``.. admonition:: Fun Fact``
+==================  ============================
+
+Avoid using other RST admonitions, as well as over-using custom admonitions. Custom admonitions should not be used for one-off or limited-use purposes. If you feel the urge to call attention to content with a specific title, consider using a section heading.
+
+Admonition Usage
+----------------
+
+Strive to use admonitions consistently. Admonitions are effective only when the enhance the readibility of a text. If admonition usage is inconsistent, then readers will learn to ignore the titles, decreasing the effectiveness of all admonitions.
```

Comment:
Typo: "they" instead of "the" following the "only when..." in the second sentence.